### PR TITLE
CDS API overview and guidelines pages

### PR DESCRIPTION
**Pull request by afzalmufti**
New info pages for CDS API overview and developer guidelines

Staging links:
https://uec-tech-standards.netlify.com/cds-api-overview.html
https://uec-tech-standards.netlify.com/cds-api-developer-guidelines.html

### CODE REVIEW

File: _pages/business_capabilities/cds-api-overview.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+sidebar: overview_sidebar
+permalink: cds-api-overview.html
+toc: true
+summary: This page is intended as an orientation guide for API developers, who are making systems compliant to the Clinical Decision Support API (CDS API)
+folder: business_capabilities
+---
+
+NHS Digital is committed to setting strong, open standards for inter-operable internet-first digital service provision. Development of standards, frameworks and Application Programming Interfaces (APIs) are the chosen route for improving data sharing and giving clinicians better access to patient information.
+
+The [Clinical Decision Support API Specification v1.0.0](https://developer.nhs.uk/apis/cds-api-1-0-0/) is the first standard published to support the interactions between a Clinical Decision Support System (CDSS) and an Encounter Management System (EMS) to support patient triage using open standards.
```
Comment (mattstibbs) [inline _pages/business_capabilities/cds-api-overview.md:12]: I would suggest a paragraph on "Who might find this useful?" would be a good thing to have

File: _data/glossary.yml [hunk 2]
```diff
@@ -1,9 +1,79 @@
...
-  definition: HL7.org is an international organisation responsible for the creation and maintenance of information and interoperability standards for healthcare.
+- keyword: 111
+  definition: The non emergency, free to call telephone number used across England for single point of entry to access Integrated Urgent Care services; also an online service for non-emergency care for the same purpose.
+- keyword: 999
+  definition: The emergency telephone number used across England to access emergency services in a medical emergency  when someone is seriously ill or injured and their life is at risk.
+- keyword: A&E
+  expansion: Accident and Emergency
+  definition: An A&E department (also known as emergency department or casualty) deals with genuine life-threatening emergencies.
+- keyword: Alpha
+  definition: A development stage in iterative delivery to build a basic working system with limited functionality used to demonstrate to users
```
Comment (mattstibbs) [inline _data/glossary.yml]: This definition needs improving a little - suggest borrowing from the GDS definition https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works

File: _data/glossary.yml [hunk 3]
```diff
@@ -1,9 +1,79 @@
...
+  expansion: Predictive Modelling of Patient Risk
+  definition: Using historical data to identify the patient-level variables that predict over-triage and under-triage for different Pathways or symptom groups. These patient-level variables can then be used to identify future service users who are at high risk of over- and under-triage.
+- keyword: POC 
+  expansion: Proof of Concept
+  definition: A proof of concept is a demonstration, the purpose of which is to verify that certain concepts or theories have the potential for real-world application. POC is therefore a prototype that is designed to determine feasibility, but does not represent deliverables.
+- keyword: Provider 
+  definition: Entity contracted by a commissioner to provide a service
+- keyword: SNOMED CT 
+  expansion: Systemised Nomenclature of Medicine (Clinical Terms). 
+  definition: A structured clinical vocabulary for use in an electronic health record. It is the most comprehensive and precise clinical health terminology product in the world.
```
Comment (mattstibbs) [inline _data/glossary.yml]: Suggest adding a mention that it is the mandated official clinical terminology for the NHS.

File: _data/glossary.yml [hunk 4]
```diff
@@ -1,9 +1,79 @@
...
+  definition: The process of determining the priority of patients' treatments based on the severity of their condition
+- keyword: UEC
+  expansion: Urgent and Emergency Care 
+  definition: An area of Health Care services that primarily deals with urgent same day outpatient contacts for patient needs such as minor illness and injuries, and involves sign-posting and treatment of patients in settings such as 111, 999, OOH (Out Of Hours) GPs, A&E and minor injuries units, or through arrangement of urgent appointments (GP and Dental).
+- keyword: UTC
+  expansion: Urgent Treatment Centres
+  definition: Medical treatment facility where patients can be treated for minor emergencies and injury.
+- keyword: WIC
+  expansion: Walk-in centre
+  definition: NHS walk-in centres offer convenient access to a range of treatments. WICs are managed by Clinical Commissioning Groups (CCGs), dealing with minor illnesses and injuries
```
Comment (mattstibbs) [inline _data/glossary.yml]: You should reference that WICs will eventually be replaced by UTCs.

File: _pages/emerging/clinical_decision_support_api.md [hunk 5]
```diff
@@ -12,4 +12,6 @@ The guide covers triage of patients by non-clinical staff and triage of patients
 
 The Implementation Guide is based on initial discovery work, which informs the 1.0.0-alpha version of the Guide. The scope is expected to increase with more discovery work and as use cases are developed. Future versions of this Guide will include those updates as they are developed. Any resources not specifically mentioned in the Guide will follow the [HL7 FHIR STU3 guidance](https://www.hl7.org/fhir/stu3/index.html).
 
-[Access the Clinical Decision Support API implementation guide](https://developer.nhs.uk/apis/cds-api/)
\ No newline at end of file
+[Access version 1.0.0 the Clinical Decision Support API implementation guide](https://developer.nhs.uk/apis/cds-api-1-0-0/)
```
Comment (mattstibbs) [inline _pages/emerging/clinical_decision_support_api.md]: **"of"** the Clinical Decision Support API....

**Review by mattstibbs — CHANGES_REQUESTED** [review]
WIthin the CDSS API FAQ section, section "FHIR Primers" - you should link out to the offical documentation on those concepts, not just attempt to define them ourselves.

**Comment by afzalmufti** [comment]
Pushed fresh commit addressing all comments with exception of Who might find this useful? paragraph. Hoping summary at the top of the page covers this and is enough until next iteration.

File: _pages/business_capabilities/cds-api-overview.md (hunk 1 above)
Comment (afzalmufti) [reply _pages/business_capabilities/cds-api-overview.md:12]: Summary at top of page: This page is intended as an orientation guide for API developers, who are making systems compliant to the Clinical Decision Support API (CDS API)

File: _data/glossary.yml (hunk 2 above)
Comment (afzalmufti) [reply _data/glossary.yml]: Amended: 
A development stage in iterative delivery where a project team takes new insights and findings from the [discovery](#discovery) phase and begins to experiment with solutions to a challenge. The alpha phase is about rapidly testing ideas to see what works and what doesn't.

Also added entries for discovery and beta:

Discovery
The first stage of development in iterative delivery where a project team analyses a problem and conducts research to understand the users, needs and journeys that need to be considered when developing a solution.

Beta
A development stage in iterative delivery where the learning from the [alpha phase](#alpha) is developed into a working service and tested with real users while minimising risk and maximising the potential to learn and iterate the service.

File: _data/glossary.yml (hunk 3 above)
Comment (afzalmufti) [reply _data/glossary.yml]: Amended:
A structured clinical vocabulary for use in an electronic health record. It is the most comprehensive and precise clinical health terminology product in the world. SNOMED CT is to be used as the single terminology in all care settings in England.

File: _data/glossary.yml (hunk 4 above)
Comment (afzalmufti) [reply _data/glossary.yml]: Amended slightly:
NHS walk-in centres offer convenient access to a range of treatments. WICs are managed by Clinical Commissioning Groups (CCGs), dealing with minor illnesses and injuries. Walk-in centres are gradually being replaced by urgent treatment centres (UTCs).

Beefed up UTC description:

A medical treatment facility where patients can be treated for minor emergencies and injury. UTCs work alongside other parts of the urgent care network including primary care, community pharmacists, ambulance and other community-based services to provide a locally accessible and convenient alternative to A&E for patients who do not need to attend hospital. Commonality of service provision under the UTC name will increasingly end the current confusing range of options and simplify the system so patients know where to go and have clarity of which services are on offer wherever they are in the country.

File: _pages/emerging/clinical_decision_support_api.md (hunk 5 above)
Comment (afzalmufti) [reply _pages/emerging/clinical_decision_support_api.md]: fixed

**Review by mattstibbs — APPROVED** [review]

**Pull request merged**